torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -1745,7 +1745,9 @@ void jitModuleToPythonCodeAndConstants(
     class_deps.add(class_type);
   }
 
-  for (const auto i : c10::irange(class_deps.size())) {
```

Comment:
what's wrong with c10::irange?

Reply:
Can we add a unit test?

Reply:
inside of the loop might make class_deps longer (somewhere appends to it). Good point on the unit test let me think